### DISCUSSION TITLE
3D Virtual Test Track for Autonomous Driving (Project 171)

### DISCUSSION THREAD

**Discussion by robertogl** [mathworks/MATLAB-Simulink-Challenge-Project-Hub#20] [Collaborate]
Contribute to the discussion by asking and/or answering questions, commenting, or sharing your ideas for solutions to [project #171](https://github.com/mathworks/MathWorks-Excellence-in-Innovation/tree/main/projects/3D%20Virtual%20Test%20Track%20for%20Autonomous%20Driving)

---
*:robot: An AI bot may reply to your comments with suggestions and guidance. It will not solve the project for you. Your comments may be processed by a third-party LLM. To direct a question to a specific person, tag them with @username.*

**Comment by Venkat8870** [comment]
hi sir I'm Venkata Sai from vnrvjiet.i'm interested in this project and I just want some suggestions before starting this project sir

**Reply by pfryscak** [reply]
Hi Venkata, What kind of suggestions are you looking for? Kindly add a few questions here.

**Reply by Venkat8870** [reply]
Sir at present I have only matlab installed on my laptop I want road runner to do my project my University is paying for matlab i don't know how to add matlab to my pc even though I have only license for matlab

**Reply by pfryscak** [reply]
Please follow [these steps](https://www.mathworks.com/content/dam/mathworks/mathworks-dot-com/images/responsive/supporting/solutions/automated-driving/roadrunner-tutorial/access-roadrunner-cwl.pdf) to install and activate RoadRunner using your Campus-Wide License. I recommend you try our self-paced [RoadRunner Tutorial](https://www.mathworks.com/solutions/automated-driving/roadrunner-tutorial.html) to learn how to design scenes with it.

**Comment by Venkat8870** [comment]
sir i dont know where to ask could you please guide me how to install road runner 2023a version sir please

**Comment by pfryscak** [comment]
In the document above there is a link: Identify your license administrator (Instructions).

**Reply by Venkat8870** [reply]
Oh thankyou sir

**Comment by harry-po** [comment]
Hello sir I am waskita from Indonesia, I am interested in this project, could you share what I should learn before starting this project? 
thankyou.

**Comment by pfryscak** [comment]
Hi Waskita! Kindly send an email to: automated-driving@mathworks.com from your university email address. In the meantime, please take a look at: https://www.mathworks.com/products/roadrunner.html.

**Comment by Hadron-JLeo** [comment]
Hello, I am Leonard from Germany.
I have read through the instructions and will be starting with this project soon.

Cheers!

**Reply by Hadron-JLeo** [reply]
Good day.

Sadly I still do not have access to the software. 
What if I were to submit a track made using Unreal Engine or Blender? I am able to export as .fbx there. @pfryscak

**Reply by Hadron-JLeo** [reply]
Nevermind, I now have access to the software. Thanks!

**Comment by Vinay8622** [comment]
Hi Sir, My name is Vinay, and pursuing a Master's in Autonomous Driving Field. I will start this project to gain practical experience in this field. 

Thank you
